docs/en/glossary/glossary.asciidoc

```diff
@@ -281,6 +287,11 @@ include::{kib-repo-dir}/glossary.asciidoc[tag=dashboard-def]
 real time. {dfeeds-cap} retrieve data from {es} for analysis.
 //Source: X-Pack
 
+[[glossary-dataset]] dataset::
+A description of ingested data and its structure for each index.
```

Comment:
A collection of data that has the same structure. The name of a dataset typically signifies its source https://github.com/elastic/ecs/blob/master/rfcs/text/0009-data_stream-fields.md